src/oss-characteristics/oss-characteristic.csproj

```diff
@@ -26,4 +26,8 @@
     <PackageReference Update="Nerdbank.GitVersioning" Version="3.1.91" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Folder Include="Properties\" />
```

Comment:
I don't see any new files in this folder in this commit.

Reply:
I removed the launchsettings.json from the folder - removed all the project launchSettings and Properties folder